10_date_timestamps_and_lies.md

```diff
@@ -0,0 +1,191 @@
+# External Data Sources - Die or die trying
+
+### Dates, Timestamps and lies
+
+I was working once with some legacy database on MySQL and one of the most common
+problems is actually dealing with dates and timestamps. But so we think.
+
+It is also not enough to know how to use the `jdbc` connector to know how to
+pull data from MySQL. You actually need to understand the mechanism of the
+connector, the database; in occurence MySQL and also Spark's in these cases.
+
+So here is the drill. I was reading some data from MySQL. Some columns are of
+type `timestamp` with a default value is "0000-00-00 00:00:00". Something like
+this :
+
```

Comment:
It could be nice to add related DDL. so reader can easily reproduce the problem. I thought about Dockerfile but it could be an overkill.